Competition2017/src/competition/subsystems/shooter/commands/ContinueShooterCommand.java

```diff
@@ -0,0 +1,32 @@
+package competition.subsystems.shooter.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.shooter.ShooterSubsystem;
+import competition.subsystems.shooter.SideShooterSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class ContinueShooterCommand extends BaseCommand {
+
+    SideShooterSubsystem sideShooter;
+
+    @Inject
+    public ContinueShooterCommand() {
+    }
+    
+    public void setSide(SideShooterSubsystem sideshooter) {
+        this.sideShooter = sideshooter;
+        this.requires(sideshooter);
+    }
+
+    @Override
+    public void initialize() {
+    
+    }
+    
+    @Override
+    public void execute() {
```

Comment:
John helped explain what's going on, I have some other ideas on how to structure this but can tackle that separately